src/theory/strings/theory_strings.cpp

```diff
@@ -889,16 +891,19 @@ void TheoryStrings::computeCareGraph(){
       }
     }
     if( has_trigger_arg ){
-      index[op].addTerm( f1, reps );
+      TypeNode ft = utils::getOwnerStringType(f1);
+      std::pair<TypeNode, Node> ikey = std::pair<TypeNode, Node>(ft, op);
```

Comment:
Just a minor simplification.

```suggestion
      std::pair<TypeNode, Node> ikey(ft, op);
```